sevenhills.html

```diff
@@ -0,0 +1,290 @@
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-12">
+                <h1 class="component-header-box component-header-box-border-lg mt-100">
+                    seven hills we die on
+                </h1>
+                <p class="font-family-base soft-top">We call them the seven hills we die on. Simply calling these
+                    “values” doesn’t cut it. All too commonly, values have come to signify mushy, sentimental ideals for
+                    which we rarely fight. As a church, we believe that if something is worth living for, it is worth
+                    dying for. We realize we can’t do everything—but what we, as a community, are called by God to do,
+                    we do with excellence</p>
+            </div>
+        </div>
+    </div>
+</section>
+
+<!-- Authenticity -->
+<section class="soft-bottom bg-blue">
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-md-7 text-center soft-ends push-top">
+                <img class="seven-hills-img-size" src="//crds-media.imgix.net//5aYjNwuHn3mSkh11bvPGd6/9634a06521d18a842e373362eb101887/death-to-religion.png">
+            </div>
+            <div class="col-md-5 content-center content-right">
+                <div class="portrait-card-content">
+                    <div class="portrait-card-arrow">
+                        <svg width="14px" height="38px" viewBox="0 0 14 38" version="1.1"
+                            xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+                            <g id="About-Crossroads" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"
+                                fill-opacity="0.35">
+                                <g id="About-Us---Final" transform="translate(-84.000000, -1891.000000)" fill="#E8E2D0"
+                                    fill-rule="nonzero">
+                                    <polygon id="ornaments-arrow-down"
+                                        points="92 1891 92 1915 98 1915 91 1929 84 1915 90 1915 90 1891"></polygon>
+                                </g>
+                            </g>
+                        </svg>
+                    </div>
+                    <p class="font-family-base-bold text-uppercase text-orange flush">the hill of:</p>
+                    <h2 class="section-header text-white flush">authenticity</h2>
+                    
+                        <p class="push-top">We believe we can’t be what God has called us to be if we play games with each other. Whether
+                            it’s what happens on stage, within our smaller communities, in serving teams or in our
+                            homes, we need to be able to share our faults and weaknesses and not fake it. That’s
+                            authenticity, just being real. (From Philippians 3:17 in the Bible</p>
```

Comment:
I don't normally pay much attention to content but it looks like you haven't closed this parenthesis?

Reply:
Ah, thanks for catching that!